lib/presentation/localizations/app_localizations.dart

```diff
@@ -70,6 +70,11 @@ class AppLocalizations {
       name: 'https');
   }
 
+  String get url {
+    return Intl.message('URL',
+        name: 'url');
+  }
```

Comment:
Can you indent like previous functions please :(

Reply:
I fixed and pushed a new commit. Thank you, please

Reply:
don't need the `URL`. Use the same `https:// `